src/preloadjs/LoadQueue.js

```diff
@@ -787,7 +787,7 @@ this.createjs = this.createjs || {};
 	 * It is recommended that loaders extend {{#crossLink "AbstractLoader"}}{{/crossLink}}. Loaders can only be added
 	 * once, and will be prepended to the list of available loaders.
 	 * @method registerLoader
-	 * @param {Class} The AbstractLoader class to add.
```

Comment:
This is doc'd this way on purpose. We do not enforce an AbstractLoader (only as an interface), and a developer could make their own that does not extend AbstractLoader. We could do this though, as it provides an idea on what the loader classes need to define as an API.